### ISSUE TITLE
Support for "Flat Config"

### ISSUE THREAD

**Issue by nagisaando** [dukeluo/eslint-plugin-check-file#41]
Hello! 

Does this plugin support [Flat config](https://eslint.org/docs/latest/use/configure/configuration-files-new)? (eslint.config.js). Since the other config files will be deprecated after ESLint v9.0.0, it would be great if we could use it with the new config. (Sorry if it is already available. I might have missed)

Currently, if I import the plugin to the new config file, it shows error as 

```
import eslintCheckFile from 'eslint-plugin-check-file';
       ^^^^^^^^^^^^^^^
SyntaxError: The requested module 'eslint-plugin-check-file' does not provide an export named 'default'
```

```
// eslint.config.js
import eslintCheckFile from 'eslint-plugin-check-file';
export default [
	...eslintCheckFile,
];
```

**Comment by dukeluo** [comment]
Currently, this plugin does not support the new "Flat Config" feature. However, I am considering adding support for it in future versions. I need some time to understand how to implement this new feature. In the meantime, any contributions via a PR to add this functionality would be greatly appreciated.

**Comment by nagisaando** [comment]
Thanks for the clarification. Yes I will take a look

**Comment by dukeluo** [comment]
@nagisaando I have released v2.8.0 to support flat config for eslint v9.0.0.

**Comment by nagisaando** [comment]
Hello @dukeluo . Thank you so much for taking time to implement it. I just tried it out, and it is working great. Thanks again!
